jupyterlab_git/git.py

```diff
@@ -16,6 +16,45 @@ def __init__(self, root_dir, *args, **kwargs):
         super(Git, self).__init__(*args, **kwargs)
         self.root_dir = os.path.realpath(os.path.expanduser(root_dir))
 
+    def config(self, **kwargs):
+        """Get or set Git options.
+        
+        If no kwargs, all options are returned. Otherwise kwargs are set.
+        """
+        response = {"code": 1}
+
+        if len(kwargs):
+            output = []
+            for k, v in kwargs.items():
+                cmd = ["git", "config", "--global", "--add", k, v]
+                p = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+                out, err = p.communicate()
+                output.append(out.decode("utf-8").strip())
+                response["code"] = p.returncode
+                if p.returncode != 0:
+                    response["command"] = " ".join(cmd)
+                    response["message"] = err.decode("utf-8").strip()
+                    return response
+
+            response["message"] = "\n".join(output).strip()
+        else:
+            cmd = ["git", "config", "--global", "--list"]
+            p = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
We don't need to check for the global configuration, right. As long as user name/email is configured at the repo or global leve, we're good. `git config user.email/name` will return the repo level user name/email or the global one if no override is set.

Reply:
You are right, I'll do that.